libtock/crt0.c

```diff
@@ -160,7 +160,6 @@ void _start(void* app_start __attribute__((unused)),
     "movs r0, r6\n"             // first arg is app_start
     "movs r1, r7\n"             // second arg is mem_start
     "bl _c_start_pic\n"
-    "bkpt #255\n"
```

Comment:
`_c_start_pic` is marked as `noreturn`, so this instruction is useless and  never reached. This change is consistent with RISC-V variant.

Reply:
_Technically_, a compiler _could_ ignore the `noreturn` attribute. This is a 2-byte hedge for a corner case of something going particularly wonky (see the comment above explaining its purpose). I'd rather _add_ this to RISC-V than remove it here.

Reply:
This looks mostly like a defensive approach. `_c_start_pic` and `_c_start_nopic` are the functions marked as `noreturn`. Most end users will never have to touch these functions. Furthermore, modern compilers are good enough to determine if a `noreturn` function has an execution path that leads to exiting the function. The only way one could return from a `noreturn` function is through inline assembly, but at that point everything is possible, including modifying the return address to point past the break point making `bkpt` and `ebreak` useless.